pkg_defs/bep_pcb_check/meta.yaml

```diff
@@ -0,0 +1,44 @@
+package:
+  name: bep_pcb_check
+  version:  {{ SKA_PKG_VERSION }}
+
+build:
+  noarch: python
+  script_env:
+    - SKA_TOP_SRC_DIR
+
+source:
+  path: {{ SKA_TOP_SRC_DIR }}/bep_pcb_check
+
+
+# the build and runtime requirements. Dependencies of these requirements
+# are included automatically.
+requirements:
+  # Packages required to build the package. python and numpy must be
+  # listed explicitly if they are required.
+  build:
+    - pip
+    - python
+    - setuptools
+    - six
+    - numpy
+    - matplotlib
+    - acis_thermal_check
```

Comment:
And I *think* everything now needs setuptools_scm in build.  Right @javierggt ?

Reply:
Oops, it looks like these new packages aren't set for setuptools_scm / ska_helpers so that previous comment may be moot for now?